src/background/background.js

```diff
@@ -0,0 +1,41 @@
+import { initDB } from './initdb.js';
+import { fetchAndStoreHistory } from './handlers/fetchAndStoreHistory.js';
+import { getMostVisitedSites } from './handlers/getMostVisitedSites.js';
+import { getLabelCounts } from './handlers/getLabelCounts.js';
+import { getCOCounts } from './handlers/getCOCounts.js';
+import { getVisitsPerHour } from './handlers/getVisitsPerHour.js';
+import { getDailyVisitCounts } from './handlers/getDailyVisitCounts.js';
+import { getCategoryTrends } from './handlers/getCategoryTrends.js';
+import { getTransitionPatterns } from './handlers/getTransitionPatterns.js';
+import { getTimeSpentPerSite } from './handlers/getTimeSpentPerSite.js';
+import { getRecencyFrequency } from './handlers/getRecencyFrequency.js';
+
+(async function init() {
+  try {
+    await initDB();
+    console.log('[background] Database initialized');
+  } catch (err) {
+    console.error('[background] Database initialization failed', err);
+  }
+})();
+
+window.fetchAndStoreHistory = fetchAndStoreHistory;
```

Comment:
same remark, if you have a single `handlers` var, you could do something like a single`window.recapHandlers = handlers;` here